recipe/meta.yaml

```diff
@@ -47,6 +46,7 @@ outputs:
         - numpy                                  # [build_platform != target_platform]
         - pybind11                               # [build_platform != target_platform]
       host:
+        - cuda-version {{ cuda_compiler_version }}  # [(cuda_compiler_version or "None") != "None"]
         - cudnn                                  # [cuda_compiler_version != "None"]
```

Comment:
Is it possible to build without cuDNN (like on ARM)? If so, we could add a selector here

For more context on ARM and cuDNN, please see issue ( https://github.com/conda-forge/cudnn-feedstock/issues/58 )

Reply:
I'm not sure. Upstream only builds cuda packages if both cuda and cudnn are found: https://github.com/davisking/dlib/blob/46e59a2174228922d19d2887756d6dbfef80dc04/dlib/CMakeLists.txt#L781-L795